src/styles/app.styl

```diff
@@ -54,6 +54,18 @@ body
     background-color rgba(0, 0, 0, .32)
     backdrop-filter saturate(124%)
 
+.flagship-app 
+    .App
+        padding-top var(--flagship-top-height)
+        padding-bottom var(--flagship-bottom-height)
+
+    .MuiDialog-root
+        margin-top var(--flagship-top-height)
+        padding-bottom var(--flagship-bottom-height)
+
+    .MuiBackdrop-root
+        opacity 0 !important
```

Comment:
If we need to target MUI classes, then I don't think we should handle that in the App. We should handle that in Cozy-UI by itself in our theme. @acezard @Ldoppea what do you think?

Reply:
@Crash-- as it is very specific to cozy-home here, I would apply the styling in cozy-home. But yeah we could also consider it as a special case scenario in MUI.

In the end it was far quicker to handle everything in cozy-home (and everything is at the same place)

Reply:
@Crash-- agree

@acezard I struggle to imagine what does `MuiBackdrop-root opacity 0` fix? Is this a backdrop that was not visible when fullscreen and that is now visible with margins?

Reply:
@Ldoppea indeed, I figured we did not want that as it made OS controls harder to see